packages/thirdweb/src/exports/wallets/smart.ts

```diff
@@ -1,5 +1,7 @@
 export { smartWallet } from "../../wallets/smart/smart-wallet.js";
 
+export { waitForUserOpReceipt } from "../../wallets/smart/lib/userop.js";
```

Comment:
only exposing this one for now - gotta do some cleanup in the inputs of the other ones before exporting